deploy/setup-project.sh

```diff
@@ -22,34 +22,56 @@ readonly KUBEDEPLOY="${PKGDIR}/deploy/kubernetes"
 readonly BIND_ROLES="roles/compute.storageAdmin roles/iam.serviceAccountUser projects/${PROJECT}/roles/gcp_compute_persistent_disk_csi_driver_custom_role"
 readonly IAM_NAME="${GCE_PD_SA_NAME}@${PROJECT}.iam.gserviceaccount.com"
 
+# Check if SA exists
+CREATE_SA=true
+SA_JSON=$(gcloud iam service-accounts list --filter="name:$IAM_NAME" --format="json")
+if [ "[]" != "$SA_JSON" ]; then
```

Comment:
nit: `then` on new-line for file consistency
or fix the other one to be on the same line